extensions/amp-bind/amp-bind.md

```diff
@@ -138,21 +143,34 @@ Check out the AMP Conf 2017 talk "[Turing complete...AMP Pages?!](https://www.yo
 <button on="tap:AMP.setState({currentAnimal: 'cat'})">Set to Cat</button>
 ```
 
-In this example, tapping the button will change the following:
+When the button is pressed:
+
+1. **State** is updated with `currentAnimal` defined as `'cat'`
+2. **Expressions** that depend on `currentAnimal` are evaluated.
+  - `'This is a ' + currentAnimal + '.'` => `'This is a ' + 'cat' + '.'` => `'This is a cat.'`
+  - `myAnimals[currentAnimal].style` => `myAnimals['cat'].style` => `'redBackground'`.
+  - `myAnimals[currentAnimal].imageUrl` => `myAnimals['cat'].imageUrl` => `/img/cat.jpg`
+3. **Bindings** that depend on the changed expressions are updated.
+  - The first `<p>` element's text will read "This is a cat."
+  - The second `<p>` element's `class` attribute will be "redBackground".
+  - The `amp-img` element will show the image of a cat.
```

Comment:
Maybe it's helpful to some to really spell this out but IMO this is a bit too verbose and makes this section more intimidating to read at a glance.